src/redux/Categories/categories.js

```diff
@@ -0,0 +1 @@
+// Redux
```

Comment:
- [ ] Kindly make sure to implement the categories actions and reducers, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Write%20your%20categories,the%20current%20state.) requirement.